CHANGELOG.md

```diff
@@ -1,5 +1,206 @@
 # Unreleased
 
+## Unreleased 9.0.0 (Breaking release)
+
+This release updates GOV.UK Prototype Kit to v3.0.0 of GOV.UK Frontend.
+
+In v3.0.0 of GOV.UK Frontend, we’ve made some important changes to [improve the accessibility of pages](https://designnotes.blog.gov.uk/2019/07/29/weve-made-the-gov-uk-design-system-more-accessible). This includes making sure that the styles, components and patterns in GOV.UK Frontend meet [WCAG 2.1 level AA](https://www.w3.org/TR/WCAG21/).
```

Comment:
@36degrees @hannalaakso This para has changed slightly to avoid duplicating 'GOV.UK Frontend' twice in the same sentence, and to add the WCAG link that we added to the Frontend release note.